app.py

```diff
@@ -48,7 +47,7 @@
 DUMMY_VALUES = {
     'principal': 'default_principal',
     'rh_account': '000001',
-    'hash': 'abcdef123456',
+    'payload_id': '1234567890abcdef',
```

Comment:
This value is not used anywhere. Currently only _principal_ and _rh_account_ are [used](https://github.com/RedHatInsights/insights-upload/blob/49c34eb30f20186fc7605e0dbbd77bcba89c5bbb/app.py#L326). This replacement is ok in scope of this PR, I’d just consider removing them in a follow-up one.